packages/oaf/src/commands/kol/spade.ts

```diff
@@ -83,6 +87,7 @@ const ITEM_SPADING_CALLS = [
       [
         "inv_spleen.php",
         {
+          pwd: true,
```

Comment:
wait, are these still plugged in? is this still how `/spade items` works?

Reply:
Yes